src/test/java/bio/terra/service/dataset/DatasetJsonConversionTest.java

```diff
@@ -146,42 +161,52 @@ public void setUp() throws Exception {
                                 .rootTable(DATASET_TABLE_NAME)
                                 .rootColumn(DATASET_COLUMN_NAME)
                                 .follow(Collections.emptyList()))))
+            .snapshotBuilderSettings(new SnapshotBuilderSettings())
             .dataProject(DATASET_DATA_PROJECT);
 
     metadataDataAccessUtils = new MetadataDataAccessUtils(null, null, null);
   }
 
   @Test
   public void populateDatasetModelFromDataset() {
+    when(snapshotBuilderService.getSnapshotBuilderSettings(DATASET_ID))
+        .thenReturn(new SnapshotBuilderSettings());
     assertThat(
-        DatasetJsonConversion.populateDatasetModelFromDataset(
+        datasetJsonConversion.populateDatasetModelFromDataset(
             dataset,
             List.of(
                 DatasetRequestAccessIncludeModel.SCHEMA,
                 DatasetRequestAccessIncludeModel.PROFILE,
+                DatasetRequestAccessIncludeModel.SNAPSHOT_BUILDER_SETTINGS,
                 DatasetRequestAccessIncludeModel.DATA_PROJECT),
             metadataDataAccessUtils,
             testUser),
         equalTo(datasetModel));
+    verify(snapshotBuilderService).getSnapshotBuilderSettings(DATASET_ID);
```

Comment:
If you use `MockitoExtension` then you don't need this `verify()`, since an unused mock would trigger a test failure.